src/ast.ls

```diff
@@ -3228,6 +3229,8 @@ UTILS =
       }
     }'''
 
+    array-from: 'Array.from || function(x){return slice$.call(x);}'
```

Comment:
I did try that, but `Function.prototype.call` needs something to bind it to `slice$` or it forgets where it came from.